velox/exec/tests/SparkAggregationFuzzerTest.cpp

```diff
@@ -45,7 +45,7 @@ int main(int argc, char** argv) {
   // Calls common init functions in the necessary order, initializing
   // singletons, installing proper signal handlers for better debugging
   // experience, and initialize glog and gflags.
-  folly::Init init(&argc, &argv);
```

Comment:
because `init` is what's marked as deprecated:
```cpp
[[deprecated("Use the RAII version Init")]] void init(
    int* argc, char*** argv, bool removeFlags = true);
[[deprecated("Use the RAII version Init")]] void init( 
```
https://github.com/facebook/folly/blob/main/folly/init/Init.h#L99C1-L102

If this is an issue for that specific test we could disable it just for that target with `target_compile_options( ... -Wno-deprecated-declarations)`. But I did not want to do that for everything.

Reply:
@assignUser, `folly::Init{&argc, &argv, true};` is causing the Init object to be destroyed immediately and not at the end of main. That is why we are seeing issues with benchmark and fuzzer.
Looking at the RAII folly change here https://github.com/facebook/folly/commit/7bf1486094cccb266e789a378d8e5f91e3cb7780#diff-fe24748541a4b73975e31bb08821bb15c783ecdd74e2057d256c4cd1b893ba92
`folly::init(&argc, &argv);` is deprecated and the RAII version is `folly::Init init(&argc, &argv);`
The later does call the RAII API.